ecl/resource_ecl_network_gateway_interface_v2_test.go

```diff
@@ -65,6 +65,10 @@ func TestAccNetworkV2GatewayInterfaceMultiGateway(t *testing.T) {
 				Config:      testAccNetworkV2GatewayInterfaceMultiGateway,
 				ExpectError: regexp.MustCompile("\"internet_gw_id\": conflicts with vpn_gw_id"),
 			},
+			resource.TestStep{
```

Comment:
`resource.TestStep` seems redundant. Since there are many other similar codes, it is optional to modify this.